wiki/Ranking/en.md

```diff
@@ -0,0 +1,42 @@
+---
+stub: true
+tags:
+  - pp
+  - leaderboards
+  - ranking
+---
+
+# Ranking
+
+**Ranking**, alternatively called *leaderboards*, is a list that sorts members of the osu! community according to certain criteria. This can also refer to one's position in such a list.
```

Comment:
```suggestion
**Ranking** (alternatively called "leaderboards") is a list that sorts members of the osu! community according to certain criteria. This can also refer to one's position in such a list.
```